.goreleaser.yaml

```diff
@@ -0,0 +1,55 @@
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - id: darwin
+    binary: '{{ .ProjectName }}-{{ .Env.DEFAULT_VERSION }}-{{ .Os }}-{{ if eq .Arch "amd64" }}x86_64{{ end }}{{ if eq .Arch "arm64" }}aarch64{{ end }}/{{ .ProjectName }}'
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -buildmode=pie
```

Comment:
I believe we have more flags than that? like `-s`?

Reply:
we might be lucky using this https://goreleaser.com/customization/universalbinaries/